languages/julia/reference/why-do-people-use-julia.md

```diff
@@ -11,14 +11,14 @@ The source for all information in this paragraph is the [Julia User & Developer
 3. Open source code is available and can be modified
 4. Multiple dispatch
 5. Solves the two language problem<sup>1</sup>
-7. Integrates well with other language(s)
-9. 1-based indexing
+6. Integrates well with other language(s)
```

Comment:
The numbering here is significant, a prettier should not mess with it.

Reply:
I've added the ignore annotation.